torch/_dynamo/variables/builtin.py

```diff
@@ -1240,6 +1240,19 @@ def call_setattr(
                         return getattr_var
 
             obj.convert_to_unspecialized(tx)
+        # FIXME (tmanlaibaatar) this is utter hack to unblock HuggingFace export
+        # Export generally doesn't want to allow mutations on objects directly,
+        # but we don't have good way to do this rn. For now, we make it an undefined
+        # behaviour and just set attributes directly on the PretrainedConfig object
+        # for now.
+        elif isinstance(obj, variables.dicts.HFPretrainedConfigVariable) and tx.export:
```

Comment:
Could we not do this ? Or at the various least, only do this on export ? if someone runs into this it would be a huge pain to debug 

cc @voznesenskym @anijain2305 re: dynamo

Reply:
This *does* have an `export` check (tx.export) - however, this is still the wrong way to do it. 

You should make a VT to reflect it and route the setattr call there.

Reply:
Should we talk about mutations in export? its a subproblem of a larger issue we have.